packages/protocol_cosmwasm/src/anchor_verifier.rs

```diff
@@ -0,0 +1,78 @@
+pub use self::anchor_verifier::AnchorVerifier;
+
+mod verifier {
```

Comment:
Is this the same `mod verifier` implementation as what is used in the mixer? Can we reuse this module?

Reply:
Re-structured the "packages/protocol_cosmwasm".
Created a new file for "verifier" module.
https://github.com/webb-tools/protocol-cosmwasm/pull/15/commits/7fb32e12cbfd028ebbdf16cdce766946753afa17

Reply:
Moved "verifier" code to "packages/protcocol_cosmwasm" for code reuse.
https://github.com/webb-tools/protocol-cosmwasm/pull/15/commits/7fb32e12cbfd028ebbdf16cdce766946753afa17